### PR TITLE
WindowsShareCopy Action

### PR DESCRIPTION
**Pull request by awholegunch**
Changes to documentation and UI strings.

### CODE REVIEW

**Comment by anwar6953** [comment]
We just need to make sure to merge release/1.4 into develop, once this PR is merged.

File: core-plugins/docs/WindowsShareCopy-action.md [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
-**numThreads:** Specifies the number of parallel tasks to execute the copy operation. Defaults to 1.
+**numThreads:** Specifies the number of parallel tasks to use when executing the copy operation; defaults to 1. (Macro-enabled)
 
-**sourceDirectory:** Specifies the NetBios directory.For example: `/source/`
+**sourceDirectory:** Specifies the NetBios directory or file. For example: `source` or `source/file.log`. (Macro-enabled)
 
-**destinationDirectory:** The valid, full HDFS destination path in the same cluster where the file or files are to be
-moved. If a directory is specified with a file sourcePath, the file will be put into that directory. If sourcePath is a
-directory, it is assumed that destPath is also a directory. HDFSAction will not catch this inconsistency.
+**destinationDirectory:** The valid full HDFS destination path in the same cluster where
```
Comment (abhinavbansal246) [inline core-plugins/docs/WindowsShareCopy-action.md:30]: Should these line breaks be at 120 characters?

**Comment by abhinavbansal246** [comment]
Minor comment otherwise LGTM

File: core-plugins/docs/WindowsShareCopy-action.md (hunk 1 above)
Comment (awholegunch) [reply core-plugins/docs/WindowsShareCopy-action.md:30]: These are docs, and are easier to read and review on GitHub if set to less than 120. Docs are set to  90 characters.

**Pull request merged**